python/ray/train/_internal/data_config.py

```diff
@@ -53,6 +53,8 @@ def configure(
         world_size: int,
         worker_handles: Optional[List[ActorHandle]],
         worker_node_ids: Optional[List[NodeIdStr]],
+        num_cpus_per_trainer: int,
+        num_gpus_per_trainer: int,
```

Comment:
```suggestion
        num_cpus_per_worker: int,
        num_gpus_per_worker: int,
```

Let's rename `trainer` to `worker`? In Ray Train, the trainer is a single actor that acts like a coordinator, and it will launch a bunch of Train workers, which execute the actual training workload.